### PR TITLE
Add patients list view for receptionist

### PR DESCRIPTION
**Pull request by anna-malizjusz**

### CODE REVIEW

File: Controllers/PatientsController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using WebClinic.Data;
+using WebClinic.Models;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace WebClinic.Controllers
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    public class PatientsController : Controller
```
Comment (adapiekarska) [inline Controllers/PatientsController.cs:19]: Why can't we use the existing Api/PatientsController?

**Review by adapiekarska — CHANGES_REQUESTED** [review]

File: Controllers/PatientsController.cs (hunk 1 above)
Comment (anna-malizjusz) [reply Controllers/PatientsController.cs:19]: Why split them into API and non-api directories?

Comment (adapiekarska) [reply Controllers/PatientsController.cs:19]: So we are not supposed to have api/gui split?

Comment (anna-malizjusz) [reply Controllers/PatientsController.cs:19]: don't know, we'll see today

Comment (anna-malizjusz) [reply Controllers/PatientsController.cs:19]: Splitting/merging gui and api will be a less nightmare than rebasing my commits with yours new features :P

Comment (adapiekarska) [reply Controllers/PatientsController.cs:19]: well, that's the beauty of git :P

**Pull request merged**